Product/component/PGallery.jsx

```diff
@@ -1,34 +1,23 @@
 import React from 'react';
 import styles from './PGallery.module.css'
-import Thumbnail from './Thumbnails.jsx'
+import Thumbnail from './Thumbnail.jsx'
 
-const Gallery = ({image, thumbnails}) => (
-    <section>
-    <div className={styles.gallery}>
-      <img className={styles.main} src={image} alt="dress"></img>
+const Gallery = ({image, changeStyle, info, stylePhotos, handleThumbnailClick}) => (
+  <div className={styles.right}>
+    <div>
+      <img className={styles.image} src={image} alt="dress"></img>
       <div className={styles.thumbnails}>
-        {thumbnails.map(img =>
-          <Thumbnail img={img} />)}
+      {stylePhotos.map(photo =>
+        <Thumbnail photo={photo.url} handleThumbnailClick={handleThumbnailClick}/>)}
```

Comment:
May want to add a key property here otherwise dev console will throw unique key per child warning